pkg/api/configsync/register.go

```diff
@@ -120,3 +139,36 @@ const (
 	// declared to be created by the reconciler.
 	NamespaceStrategyExplicit NamespaceStrategy = "explicit"
 )
+
+var supportedAuthTypes = map[SourceType][]string{
```

Comment:
Minor nit, but perhaps this could be written as a function to guarantee immutability

Reply:
SG. done